### PR TITLE
Events updates - segmented sessions/rooms & tickets

### PR DESCRIPTION
**Pull request by david-cahill**
ref https://github.com/CoderDojo/community-platform/issues/446
ref https://github.com/CoderDojo/community-platform/issues/401

### CODE REVIEW

File: lib/get-event.js [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-    var id = args.id;
-
-    events.load$(id, function(err, event) {
-        if (err) {
-            return callback(err);
-        }
-
-        callback(null, event);
-    });
+  events.load$(id, function (err, event) {
```
Comment (dberesford) [inline lib/get-event.js]: replace cb function with 'callback'

File: lib/ticket-types.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+function ticketTypes(args, callback) {
+  var ticketTypes = [
+    {name: 'ninja', title: 'Ninja'},
+    {name: 'parent-guardian', title: 'Parent/Guardian'},
+    {name: 'mentor', title: 'Mentor'},
+    {name: 'other', title: 'Other'}
+  ];
+  return callback(null, ticketTypes);
```
Comment (dberesford) [inline lib/ticket-types.js]: wrap in a `setImmediate`

File: lib/ticket-types.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+function ticketTypes(args, callback) {
+  var ticketTypes = [
+    {name: 'ninja', title: 'Ninja', tooltip: 'Select Ninja to create tickets for attendees under 13 or over 13.'},
+    {name: 'parent-guardian', title: 'Parent/Guardian', tooltip: 'Select Parent/Guardian to create tickets for parents/guardians of the attendees.'},
+    {name: 'mentor', title: 'Mentor', tooltip: 'Select Mentor to create tickets for Mentors.'},
+    {name: 'other', title: 'Other', tooltip: 'Select Other to create tickets for unlisted types e.g. laptops.'}
+  ];
+  return callback(null, ticketTypes);
```
Comment (dberesford) [inline lib/ticket-types.js]: wrap in setImmediate

**Comment by dberesford** [comment]
Done with review, looks good.

**Comment by tangentfairy** [comment]
:+1: :godmode:

**Pull request merged**